docs/core/entity/media-player.md

```diff
@@ -53,6 +53,20 @@ and are combined using the bitwise or (`|`) operator.
 | `VOLUME_SET`        | Entity volume can be set to specific levels.                       |
 | `VOLUME_STEP`       | Entity volume can be adjusted up and down.                         |
 
+## States
+
+The state of a media player can take the following possible values.
```

Comment:
Long term I'd like us to implement an enum to validate the possible states and make that more clear in the source, similar to HVAC mode for climate entities.

Reply:
Yup. I just get we should get the expected states in writing anyway.